picard/ui/options/interface.py

```diff
@@ -209,14 +219,15 @@ def load(self):
         self.ui.builtin_search.setChecked(config.setting["builtin_search"])
         self.ui.use_adv_search_syntax.setChecked(config.setting["use_adv_search_syntax"])
         self.ui.quit_confirmation.setChecked(config.setting["quit_confirmation"])
-        self.ui.use_system_theme.setChecked(config.setting["use_system_theme"])
         current_ui_language = config.setting["ui_language"]
         self.ui.ui_language.setCurrentIndex(self.ui.ui_language.findData(current_ui_language))
         self.ui.filebrowser_horizontal_autoscroll.setChecked(config.setting["filebrowser_horizontal_autoscroll"])
         self.ui.starting_directory.setChecked(config.setting["starting_directory"])
         self.ui.starting_directory_path.setText(config.setting["starting_directory_path"])
         self.populate_action_list()
         self.ui.toolbar_layout_list.setCurrentRow(0)
+        current_theme = UiTheme(config.setting["ui_theme"])
```

Comment:
Since we removed previous code (with defaults), we need to ensure value read from config file is a valid `UiTheme`, else we'll have an exception raised.

I think the simplest would be to add a default value to the enum, use Python 3.6+ `Enum._missing_`:

```python
class UiTheme(Enum):
    DEFAULT = 'default'
    DARK = 'dark'
    LIGHT = 'light'
    SYSTEM = 'system'

    def __str__(self):
        return self.value

    @classmethod
    def _missing_(cls, value):
        return cls.DEFAULT
```

To test: stop Picard, change `ui_theme` value in config file to something invalid, start Picard

Reply:
<details>
<summary>
Best circle is a closed one xD
</summary>
https://www.youtube.com/watch?v=ZvdXNx-dtQk
</details>